en/micro-integrator/mkdocs.yml

```diff
@@ -336,7 +336,9 @@ nav:
         - File Connector:
           - 'File Connector Example': 'references/connectors/file-connector/file-connector-example.md'
         - Kafka Connector:
-          - 'kafka-connector-configuration.md': 'references/connectors/kafka-connector/kafka-connector-configuration.md'      
+          - 'kafka-connector-configuration.md': 'references/connectors/kafka-connector/kafka-connector-configuration.md'
+          - 'kafka-connector-configuration.md': 'references/connectors/kafka-connector/kafka-connector-producer-example.md'
+          - 'kafka-connector-configuration.md': 'references/connectors/kafka-connector/kafka-inbound-endpoint-example.md'
```

Comment:
The formats for these are wrong. It is like this:

"- 'Page Title': 'references/connectors/some-connector/some-connector-example.md'

Currently it will read as if the .md file name is the page title.

Reply:
Updated according to the explanation.